### ISSUE TITLE
Error setting up entry iCloud3 for device_tracker

### ISSUE THREAD

**Issue by theMartianAlien** [gcobb321/icloud3_v3#282]
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 444, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/device_tracker/config_entry.py", line 46, in async_setup_entry
    return await component.async_setup_entry(entry)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 183, in async_setup_entry
    raise ValueError("Config entry has already been setup!")
ValueError: Config entry has already been setup!

I am getting this error after i updated to the newest version.

**Comment by gcobb321** [comment]
1. What version did you just install and when. There are several 'newest versions.v3.0.rc10.4 on the icloud3 regular site as a prerelease and v3.0 on the icloud3 v3 development site.
2. When does it occur? When you start HA? When you add the icloud3 integration? When you add it after it has already been set up?
3. Does it still operate and track devices?
4. Are there any errors in the Event Log?
5. Can you do an _Event Log > Actions > Export Event Log_ and send me the file it creates? Can you send me icloud3-0.log file? Email to geekstergary@gmail.com so I can see what might be going on.
Thanks and I'll take a look at the files.

**Comment by theMartianAlien** [comment]
I checked my logs, it looked like i had both installed but the one i was using was development.

The errors appeared after i updated to the newest version of the development repositories. Then restarted the HA

That's the problem it stopped tracking location, it was telling me i was at home for the weekend when even if i was not.

The only errors in the HA event log is that one.

I tried retrieving the exported logs but i can't find it.

I deleted the repository and will try to reupload maybe that's the reason why i could not track my location properly will update you when things break again. 

Sorry for the trouble
